src/pages/likes/Likes.jsx

```diff
@@ -0,0 +1,38 @@
+import { useDocTitle } from "../../hooks/useDocTitle";
```

Comment:
suggestion: put `index` file in hooks and export everything from there. This will make your imports cleaner